xc/xc7/toolchain_wrappers/symbiflow_write_bitstream

```diff
@@ -81,7 +83,5 @@ fi
 FRAMES="${BIT%.*}.frames"
```

Comment:
You have some old variables lying around

Reply:
Done